raiden/tests/unit/test_pfs_integration.py

```diff
@@ -774,37 +758,7 @@ def test_query_paths_with_unrecoverable_pfs_error(query_paths_args):
         )
 
 
-def test_query_paths_with_insufficient_payment(query_paths_args, valid_response_json, pfs_max_fee):
-    " After an insufficient payment response, we retry only if we are below our maximum fee. "
-    insufficient_payment = [dict(error_code=PFSError.INSUFFICIENT_SERVICE_PAYMENT.value)] * 2
-    assert_failed_pfs_request(
-        query_paths_args,
-        insufficient_payment,
-        expected_requests=1,
-        exception_type=ServiceRequestIOURejected,
-    )
-
-    query_paths_args["service_config"]["pathfinding_fee"] = pfs_max_fee
-    assert_failed_pfs_request(
-        query_paths_args,
-        insufficient_payment,
-        expected_requests=1,
-        exception_type=ServiceRequestIOURejected,
-    )
-
-    query_paths_args["service_config"]["pathfinding_fee"] = int(pfs_max_fee / 2)
-    assert_failed_pfs_request(
-        query_paths_args,
-        insufficient_payment,
-        expected_requests=2,
-        exception_type=ServiceRequestIOURejected,
-    )
-
-    # second attempt not rejected
-    insufficient_payment[1] = valid_response_json
-    assert_failed_pfs_request(
-        query_paths_args, insufficient_payment, [400, 200], expected_success=True
-    )
+# TODO create tests for sufficient / insufficient payments
```

Comment:
Can you create an issue to track this as well?

Reply:
https://github.com/raiden-network/raiden/issues/4100